examples/components/mysql/main.jsonnet

```diff
@@ -0,0 +1,12 @@
+local statefulset = import "./statefulset.jsonnet";
```

Comment:
could you please move all files inside `example` to `example/kubernetes-mysql` or something like that.

This way we can add more examples (like terraform) later.

Reply:
I am not sure I understand what you mean. Can I do this after the merge?